plugins/modules/ndo_tenant_mld_snooping_policy.py

```diff
@@ -0,0 +1,359 @@
+#!/usr/bin/python
+# -*- coding: utf-8 -*-
+
+# Copyright: (c) 2024, Akini Ross (@akinross) <akinross@cisco.com>
+
+# GNU General Public License v3.0+ (see LICENSE or https://www.gnu.org/licenses/gpl-3.0.txt)
+
+from __future__ import absolute_import, division, print_function
+
+__metaclass__ = type
+
+ANSIBLE_METADATA = {"metadata_version": "1.1", "status": ["preview"], "supported_by": "community"}
+
+DOCUMENTATION = r"""
+---
+module: ndo_tenant_mld_snooping_policy
```

Comment:
I noticed that IGMP and MLD snooping policies are very similar.
Would there be any benefit in combining the two into one module?
cc: @anvitha-jain

Reply:
Would keep them separate, but they are indeed very similar